helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/client/HelpDeskPublicApiClient.java

```diff
@@ -0,0 +1,118 @@
+package org.symphonyoss.symphony.bots.helpdesk.bot.client;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+import org.symphonyoss.symphony.bots.helpdesk.bot.config.HelpDeskBotConfig;
+import org.symphonyoss.symphony.pod.model.CompanyCert;
+import org.symphonyoss.symphony.pod.model.CompanyCertDetail;
+
+import java.io.IOException;
+import java.net.HttpCookie;
+import java.util.List;
+
+/**
+ * Provides acess to public APIs of a POD.
+ * Created by campidelli on 03/12/17.
+ */
+@Service
+public class HelpDeskPublicApiClient {
+
+  @Autowired
+  HelpDeskBotConfig config;
```

Comment:
You might define this field as private and move it below static fields.

Reply:
Done.